src/moin/scripts/maint/modify_item.py

```diff
@@ -35,6 +36,17 @@ def run(self, name, meta_file, data_file, revid):
         meta = json.dumps(dict(rev.meta), sort_keys=True, indent=2, ensure_ascii=False)
         with open(meta_file, 'w', encoding='utf-8') as mf:
             mf.write(meta)
+
+        if 'charset' in rev.meta['contenttype']:
+            # input data will have \r\n line endings, output will have platform dependent line endings
+            charset = rev.meta['contenttype'].split('charset=')[1]
+            data = rev.data.read().decode(charset)
+            lines = data.splitlines()
+            lines = '\n'.join(lines)
+            with open(data_file, 'w', encoding=charset) as df:
+                df.write(lines)
+            return
```

Comment:
maybe open a ticket to refactor this in a general-use utility function. it will likely be needed at a lot of places.

(ticket because some of the recent changes in the py3 branch are NOT related to py3 porting, but rather are generic development activity)